<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-5I8QE3D" failures="1" tests="2" name="runner.test_runner" time="29.023" errors="0" timestamp="3 Nov 2022 11:36:31 GMT">
  <testcase classname="runner.test_runner" name="runScenario" time="13.170"/>
  <testcase classname="runner.test_runner" name="runScenario" time="15.853">
    <failure message="expected [SkillRary Courses] but found [SkillRary Ecommerce]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [SkillRary Courses] but found [SkillRary Ecommerce]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at definition.step_definition.i_should_see_the_username_as(step_definition.java:60)
	at âœ½.I should see the username as "SkillRary Ecommerce"(file:///C:/Users/LENOVO/Documents/workspace-spring-tool-suite-4-4.16.0.RELEASE/cucumber2/src/test/resources/login/login.feature:11)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- runner.test_runner -->
